posthog/utils.py

```diff
@@ -162,6 +162,7 @@ def render_template(template_name: str, request: HttpRequest, context=None) -> H
     template = get_template(template_name)
     try:
         context["opt_out_capture"] = request.user.team.opt_out_capture
+        context["js_posthog_apikey"] = request.user.team.api_token
```

Comment:
As we have some React frontend views that can be loaded unauthenticated (e.g. `preflight/` or `signup/`), I think this will fail when the user is unauthenticated, `request.user` will be an `AnonymousUser` which won't have a `team` attribute.

Reply:
That's already handled by the try: catch.